frontend/src/scenes/insights/Insights.js

```diff
@@ -263,35 +267,63 @@ function _Insights() {
                                     }
                                     headStyle={{ backgroundColor: 'rgba(0,0,0,.03)' }}
                                 >
-                                    {showErrorMessage ? (
-                                        <ErrorMessage />
-                                    ) : showTimeoutMessage ? (
-                                        <TimeOut isLoading={isLoading} />
-                                    ) : featureFlags['remove-shownas'] ? (
-                                        {
-                                            [`${ViewType.TRENDS}`]: <TrendInsight view={ViewType.TRENDS} />,
-                                            [`${ViewType.STICKINESS}`]: <TrendInsight view={ViewType.STICKINESS} />,
-                                            [`${ViewType.LIFECYCLE}`]: <TrendInsight view={ViewType.LIFECYCLE} />,
-                                            [`${ViewType.SESSIONS}`]: <TrendInsight view={ViewType.SESSIONS} />,
-                                            [`${ViewType.FUNNELS}`]: <FunnelInsight />,
-                                            [`${ViewType.RETENTION}`]: <RetentionContainer />,
-                                            [`${ViewType.PATHS}`]: <Paths />,
-                                        }[activeView]
-                                    ) : (
-                                        {
-                                            [`${ViewType.TRENDS}`]: <TrendInsight view={ViewType.TRENDS} />,
-                                            [`${ViewType.SESSIONS}`]: <TrendInsight view={ViewType.SESSIONS} />,
-                                            [`${ViewType.FUNNELS}`]: <FunnelInsight />,
-                                            [`${ViewType.RETENTION}`]: <RetentionContainer />,
-                                            [`${ViewType.PATHS}`]: <Paths />,
-                                        }[activeView]
-                                    )}
+                                    <div>
+                                        {lastRefresh && (
```

Comment:
not sure if it's intended to remove the label when it was recently refreshed but the UI was flashing in and out with this component. It would show the note for a brief second and then it would refresh and disappear

Reply:
@EDsCODE Fixed!

Reply:
We also need to clear this variable when you switch tabs. Otherwise we get the bug below
<img width="1037" alt="Screen Shot 2021-02-02 at 11 59 21 AM" src="https://user-images.githubusercontent.com/13127476/106636265-c103b700-654f-11eb-91b0-ce54ee52c27c.png">